### DISCUSSION TITLE
wsjtx ---> not1mm

### DISCUSSION THREAD

**Discussion by ik3ocd** [mbridak/not1mm#134] [General]
good morning Mr. Michael. sorry to bother you again, but it is not clear to me how to communicate wsjtx with not1mm, I did some tests from wsjtx the chosen callsign is not transferred to not1mm, I do not understand where I am going wrong, from your manual I did as said but without results, if possible a better help in this regard, on group configuration there is no possible action, maybe it is not possible?, also I still report with inserted callsign the attached name does not appear. I hope for your feedback and sorry again on these requests.
in use on my system is linux fedora 40 for the rest not1mm works well, my interest is more on the contest system in digi mode.
kind regards flavio ik3ocd

![server ](https://github.com/user-attachments/assets/d67319e5-31a8-4f61-9c5d-37f51c9b4ef8)

**Comment by mbridak** [comment]
@ik3ocd Hello Flavio. Nothing has to be changed in not1mm for it to get reports from wsjt-x.
not1mm listens on the wsjt-x default multicast address 224.0.0.1:2237 for the qso traffic.

In wsjt-x go to File -> Settings -> Reporting. Screen shot below.

![image](https://github.com/user-attachments/assets/2aff9c9c-b95c-40d0-a2c3-963aa1141e19)


As far as the name appearing, was the contest you were on at the time General Logging?

I get the result below when entering your call and then pressing space ot tab to go to the next field.

![image](https://github.com/user-attachments/assets/954b24b6-c8f9-41aa-8f90-cb144eeec580)

Not sure why it's not working for you.

**Comment by ik3ocd** [comment]
hi Micheal, thanks for the reply for wsjtx this works for me now, the
problem of the name after inserting call remains, the password for hamqth
is correct, yes my qso start is on general logging, and here I don't
understand why it doesn't want to work? maybe some bad management by linux
Fedora? I'll try in case Ubuntu linux or linux Mint must work somehow.
thanks again for your patience and see you soon Michael 73 de ik3ocd Flavio.

Il Ven 13 Set 2024, 20:43 Michael Bridak ***@***.***> ha
scritto:

**Comment by ik3ocd** [comment]
Hi Michael, I managed to understand what was happening with the callsign
and name when entering a QSO, unfortunately in the world of passwords,
access codes, I kept putting an old password saved on my pass sheet
forgetting to update the list, I realized this by visiting the saved
passwords on Google, sorry for my requests, my fault, thanks for your work
to the Linux community, a request when it will be possible to add the IARU
reg 1 VHF contest. Bye and 73'

Il Ven 13 Set 2024, 23:13 flavio visentin ***@***.***> ha scritto:
